crates/matrix-sdk/src/event_cache/mod.rs

```diff
@@ -496,6 +504,18 @@ impl EventCacheInner {
             }
         }
     }
+
+    async fn insert_or_update_relationship(&self, orig_ev_id: &EventId, related_ev_id: &EventId) {
```

Comment:
nit: please fully spell out `event_id` for consistency, and `original` too

(I for one would call these two parameters `original` and `related`, because including the parameter's type is spurious, since the parameter type is next to the variable name, but that's my preference :))

Reply:
I'm removing this code so it shouldn't be needed anymore.